### DISCUSSION TITLE
Plotman support pool plotting on May 17th

### DISCUSSION THREAD

**Discussion by vinayakr** [ericaltendorf/plotman#445] [General]
Are there currently plans for Plotman to support the new pool plotting when pools become available on May 17th?

**Comment by altendky** [comment]
I haven't looked into this.  What does it involve?  More than passing through the `-p`/`--pool_public_key` parameter?

https://github.com/ericaltendorf/plotman/blob/1876d4cbda73fb38584f95113a00f01af1e21704/src/plotman/resources/plotman.yaml#L125
https://github.com/ericaltendorf/plotman/blob/e2595074491622618526fd9627adf74d2c7fad5f/src/plotman/manager.py#L125-L127

**Reply by vinayakr** [reply]
I guess I'm not sure because they haven't released the pooling code yet. It sounds like they are releasing everything on Monday which seems weird to me. I posted here hoping someone would know and I could send a pull request if there are changes that need to be made.

**Reply by vinayakr** [reply]
Apparently I am not the only one not seeing any code for the pooling, people are discussing that here. https://github.com/Chia-Network/chia-blockchain/issues/3479. Bram is however saying pooling is coming on the 17th https://twitter.com/bramcohen/status/1390407484179763202

**Reply by thomann061** [reply]
https://twitter.com/bramcohen/status/1393226565455015942?s=21

It got pushed back till end of May.

**Reply by vinayakr** [reply]
Of course it did :-)
